### PR TITLE
feat: add new endpoint

### PR DESCRIPTION
**Pull request by christyjacob4**
## What does this PR do?

Add a new endpoint to return the currently active runtimes for the project. 

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

Added Tests to verify the structure and number of documents 

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)
NA

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

YES

### CODE REVIEW

**Review by lohanidamodar — APPROVED** [review]
LGTM

File: app/controllers/api/functions.php [hunk 1]
```diff
@@ -119,6 +119,35 @@
...
+    ->label('sdk.namespace', 'functions')
+    ->label('sdk.method', 'listRuntimes')
+    ->label('sdk.description', '/docs/references/functions/list-runtimes.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_RUNTIME_LIST)
+    ->inject('response')
+    ->action(function ($response) {
+        /** @var Appwrite\Utopia\Response $response */
+        /** @var Utopia\Database\Database $dbForInternal */
```
Comment (TorstenDittmann) [inline app/controllers/api/functions.php:136]: This comment is not needed.

File: tests/e2e/Services/Functions/FunctionsCustomServerTest.php [hunk 2]
```diff
@@ -795,4 +795,27 @@ public function testCreateCustomExecution()
...
+    public function testGetRuntimes()
+    {
+
+        $runtimes = $this->client->call(Client::METHOD_GET, '/functions/runtimes', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()));
+
+        $this->assertEquals(200, $runtimes['headers']['status-code']);
+        $this->assertEquals(12, $runtimes['body']['sum']);
```
Comment (TorstenDittmann) [inline tests/e2e/Services/Functions/FunctionsCustomServerTest.php:808]: In CI we only have 1 runtime 👍🏻 

For it to work locally you can check if its a positive number.

Comment (christyjacob4) [reply tests/e2e/Services/Functions/FunctionsCustomServerTest.php:808]: Oh Got it 👍

File: app/controllers/api/functions.php (hunk 1 above)
Comment (christyjacob4) [reply app/controllers/api/functions.php:136]: Will do 🙂

**Review by TorstenDittmann — APPROVED** [review]
lgtm 👍🏻 

But to stay consistent - do we might wanna add an endpoint to get a single runtime and not only the list?

**Review by eldadfux — APPROVED** [review]

**Comment by eldadfux** [comment]
> But to stay consistent - do we might wanna add an endpoint to get a single runtime and not only the list?

@TorstenDittmann I would leave it for the future

**Review by kodumbeats — APPROVED** [review]

**Pull request merged**